app/controllers/posts_controller.rb

```diff
@@ -12,6 +9,14 @@ def index
     @posts = Post.all
   end
 
+  def show
+    @post= Post.find(params[:post_id])
+  end
+
```

Comment:
Why is the new method so far down in the script? I thought it was needed at the top to first establish a new post object which could then be fleshed out in the create method with the values in params. Would love an explanation of this if you have one.

Reply:
I think the official ordering is index, show, new, edit, create, update, destroy - I will change the order now :)